src/main/java/com/carrotcreative/cream/cache/CacheManager.java

```diff
@@ -40,6 +43,22 @@ private CacheManager(Context context)
         mRootDir = context.getCacheDir();
     }
 
+    public void setLruCacheSize(int lruCacheSize)
```

Comment:
enable sounds more like a boolean thing - `setLruCacheSize` sounds good to me - however I will be willing to change it to enable if you think that is more appropriate.